UI/PlayerListWnd.cpp

```diff
@@ -320,7 +320,7 @@ namespace {
                 ErrorLogger() << "PlayerDataPanel::Render couldn't get client app!";
                 return;
             }
-            if (m_player_id != app->PlayerID()) {
+            if (m_empire_id != app->EmpireID()) {
```

Comment:
Is the player ID actually incorrect here?

Reply:
I suppose it doesn't change anything as I tested with observers and moderators. Maybe better also test for `ALL_EMPIRES`.